frontend/src/components/StatusPage/StatusPageDetailData.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { formatDistanceToNow, parseISO } from "date-fns";
+import { Address, Status, Part } from "solo-types";
+
+interface StatusPageDetailDataProps {
+  address: Address[];
+  status: Status[];
+  part: Part[];
```

Comment:
Prop naming feels weird. Feels like a singular thing when there arrays of stuff.

Reply:
After looking at this component I only think `status` should be an array and I think the same should change to something like `statuses` or `statusArray`

Reply:
@lit-dds changed the key names of the data in the api response, and an explicit conversion step from the structure of the api response to a more frontend focused data structure was added.